lib/uniris/application.ex

```diff
@@ -54,7 +54,8 @@ defmodule Uniris.Application do
       SelfRepairSupervisor,
       WebSupervisor,
       {Bootstrap, port: port, transport: transport},
-      {Task.Supervisor, name: Uniris.TaskSupervisor}
+      {Task.Supervisor, name: Uniris.TaskSupervisor},
+      Uniris.Oracles.Supervisor
```

Comment:
I think Oracle should start after bootstrap, as we may want to load from DB some previously stored Oracle transaction to fill an internal state